src/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 
 <body>
-  <header class="divider" id="main-header"></header>
+  <header class="divider" id="header"></header>
```

Comment:
the `loadHeaderFooter` function is looking for `#header` so changing these from `main-header` to `header` fixed that